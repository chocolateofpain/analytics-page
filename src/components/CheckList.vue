<template>
  <tbody>
    <td colspan="6">
    <CheckItem 
      v-for="(item, index) in checks" :key='index' v-bind:checkItemData="item"
    />
    </td>
  </tbody>
</template>

<script>
import CheckItem from './CheckItem.vue'

export default {
  name: 'CheckItemList',
  components: {
    CheckItem,
  },
  props: [
    "checks"
  ],
}
</script>
