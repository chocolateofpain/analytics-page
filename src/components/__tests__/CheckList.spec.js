import { mount } from '@vue/test-utils'
import CheckList from '../CheckList.vue'

describe('CheckList', () => {
  it('should be defined', () => {
    expect(CheckList).toBeDefined()
  })

  describe('snapshots', () => {
    it('should render component', () => {
      const component = mount(CheckList)
      expect(component).toMatchSnapshot()
    })
  })
})
