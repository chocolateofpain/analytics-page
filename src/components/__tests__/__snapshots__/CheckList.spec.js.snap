// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckList snapshots should render component 1`] = `
<tbody>
  <td colspan="6"></td>
</tbody>
`;
